src/test/test.cpp

```diff
@@ -191,6 +195,10 @@ getTestConfig(int instanceNumber, Config::TestDbMode mode)
         // only spin up a small number of worker threads
         thisConfig.WORKER_THREADS = 2;
         thisConfig.QUORUM_INTERSECTION_CHECKER = false;
+
+#ifdef BEST_OFFER_DEBUGGING
+        thisConfig.BEST_OFFER_DEBUGGING_ENABLED = !gBestOfferDebuggingDisabled;
```

Comment:
If we're running tests this should always be `true`. No need for command line configuration.

Reply:
Done.